nitro-protocol/src/transactions.ts

```diff
@@ -8,7 +8,7 @@ import {getStateSignerAddress, SignedState} from './signatures';
 // CONSTANTS
 export const MAGIC_ADDRESS_INDICATING_ETH = constants.AddressZero;
 export const MAX_TX_DATA_SIZE = 128 * 1024; // (bytes) https://github.com/ethereum/go-ethereum/blob/f59ed3565d18c1fa676fd34f4fd41ecccad707e8/core/tx_pool.go#L51
-export const NITRO_MAX_GAS = 6_000_000; // should be below the block gas limit, which can change over time and is generally different on mainnet, testnet and ganache.
+export const NITRO_MAX_GAS = 16_000_000; // should be below the block gas limit, which can change over time and is generally different on mainnet, testnet and ganache.
```

Comment:
Could you explain this increase from 6M to 16M?

Reply:
I've seen block gas limit has increased on all chains, but ganache. This change should be undone.

Also why have you put gas limit as 6 mil, not 6_721_975 as hardcoded in ganache?

Reply:
> Also why have you put gas limit as 6 mil, not 6_721_975 as hardcoded in ganache?

I struggle to remember exactly :-P. I think the idea is that we want to assert that transactions in nitro/satp do not consume excessive gas, where excessive means "getting near the block gas limit". As I recall, getting near the limit is enough to make the transaction unattractive to miners. So we picked a value well below the limit on most test nets (ganache is a bit of an exception because there is no gas auction to compete in). 

Anyways, probably not super important -- I suggest we just leave this value as is (unless it causes problems). We can revisit if needs be.